nptl/allocatestack.c

```diff
@@ -949,6 +949,10 @@ attribute_hidden
 __nptl_setxid (struct xid_command *cmdp)
 {
   int result;
+
+  // Lind: we fake any setxid calls to 0 but dont actually set them
+  return 0;
+
```

Comment:
Please comment out the code after this in the function so we don't get a dead code warning